code/src/components/Compass.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Lottie from 'react-lottie';
+import compass from '../lotties/72324-compass-element-animation';
```

Comment:
Really nice that you used the lottie in your own way, not just for the loader. I did a little bit of research about the errors we were getting with react-lottie. It turns out it's actually not been updated in more than 3 years, and there's been some changes in react that have made some 'unsafe' hooks outdated. But react-lottie hasn't been updated in response so it's still using these 'unsafe' hooks. I haven't tried it yet, but if you want to use lotties again, I would look into maybe using the react-lottie-player package instead: https://www.npmjs.com/package/react-lottie-player. Like I said, I've not used it yet, but here's some info: https://github.com/chenqingspring/react-lottie/issues/100